core/src/main/java/io/fabric8/maven/core/util/Constants.java

```diff
@@ -22,4 +22,6 @@
 public class Constants {
     public static final String RESOURCE_SOURCE_URL_ANNOTATION = "maven.fabric8.io/source-url";
     public static final String RESOURCE_APP_CATALOG_ANNOTATION = "maven.fabric8.io/app-catalog";
+    public static final String SPRING_BOOT_MAVEN_PLUGIN_GA = "org.springframework.boot:spring-boot-maven-plugin";
```

Comment:
was just trying to avoid copying and pasting strings all over the place. Happy to revert if your re-ordering idea turns out to work